### PR TITLE
Artifact retain ppd

### PR DESCRIPTION
**Pull request by josenavas**
Modifies the patch to keep the preprocessed data id as the artifact id.

### CODE REVIEW

File: qiita_db/support_files/patches/33.sql [hunk 1]
```diff
@@ -140,6 +140,13 @@ ALTER TABLE qiita.artifact ADD CONSTRAINT fk_artifact_visibility FOREIGN KEY ( v
 ALTER TABLE qiita.artifact ADD CONSTRAINT fk_artifact_software_command FOREIGN KEY ( command_id ) REFERENCES qiita.software_command( command_id )    ;
 ALTER TABLE qiita.artifact ADD CONSTRAINT fk_artifact_data_type FOREIGN KEY ( data_type_id ) REFERENCES qiita.data_type( data_type_id )    ;
 
+-- We need to keep the old preprocessed data id for the artifact id due
```
Comment (antgonza) [inline qiita_db/support_files/patches/33.sql]: Could you rewrite, a few misspellings?

**Comment by antgonza** [comment]
One minor comment.

**Comment by mortonjt** [comment]
I don't see any outstanding problems.

**Comment by antgonza** [comment]
Thanks, will merge once tests pass.

**Pull request merged**